packages/dicomImageLoader/src/decodeImageFrameWorker.ts

```diff
@@ -353,16 +378,16 @@ function scaleImageFrame(imageFrame, targetBuffer, TypedArrayConstructor) {
  * callbackFn that is called with the results.
  */
 export async function decodeImageFrame(
-  imageFrame,
-  transferSyntax,
-  pixelData,
-  decodeConfig,
-  options,
-  callbackFn
-) {
+  imageFrame: CoreTypes.IImageFrame,
+  transferSyntax: string,
+  pixelData: ByteArray,
+  decodeConfig: LoaderDecodeOptions,
```

Comment:
You can read the TsuidConfigOptions here for the specific transferSyntaxUID, then pass it in generally below, eg
const tsuidConfigOptions = decodeConfig?.[transferSyntax]
Then other options can be passed in the future.